server/api/rounds.py

```diff
@@ -298,8 +306,8 @@ def calculate_risk_measurements(election: Election, round: Round):
             p_values, is_complete = ballot_polling.compute_risk(
                 election.risk_limit,
                 sampler_contest.from_db_contest(contest),
-                cumulative_contest_results(contest),
```

Comment:
can you remove `cumulative_contest_results` now, or is it still being used elsewhere?

Reply:
I will check.

Reply:
Nope, I removed it.